src/tools/rbd/action/Snap.cc

```diff
@@ -412,7 +412,7 @@ int execute_unprotect(const po::variables_map &vm) {
 void get_rename_arguments(po::options_description *positional,
                           po::options_description *options) {
   at::add_snap_spec_options(positional, options, at::ARGUMENT_MODIFIER_SOURCE);
-  at::add_snap_spec_options(positional, options, at::ARGUMENT_MODIFIER_DEST);
+  at::add_snap_rename_dest_snap_option(options);
```

Comment:
Ditto for this -- ```add_snap_rename_dest_snap_option``` should live in this file since it isn't reused.  You have also removed the ability to specify the destination snap name as a positional argument.